lib/sen_tweet_web/templates/page/index.html.eex

```diff
@@ -1 +1,56 @@
-<div id="messages"></div>
+<div id="messages-container">
+
+  <!-- Tweet template -->
+  <article id="tweet-message" class="message" style="display: none;">
+    <div id="header" class="message-header">
+      <p id="sentiment"></p>
+      <!--<p id="score"></p>-->
```

Comment:
remove?

Reply:
Overlooked some of this stuff. Regarding null scores, it should be easy enough to return "" when the score is nan but there will be boilerplate. Good use for a function wrapper.. if I can figure out how to do that in js

Reply:
Why not Elixir ? :feelsgood: